modules/aspect_rules_js/1.0.0-rc.2/presubmit.yml

```diff
@@ -0,0 +1,12 @@
+bcr_test_module:
+  module_path: "e2e/bzlmod"
```

Comment:
it runs this in the rules_js repo?

Reply:
The real bcr runs this, yeah. Not our fork.